.htaccess_live

```diff
@@ -1,357 +0,0 @@
-<Files *.ss>
-	Order deny,allow
-	Deny from all
-	Allow from 127.0.0.1
-</Files>
-
-<Files web.config>
-	Order deny,allow
-	Deny from all
-</Files>
-
-<IfModule mod_rewrite.c>
-	RewriteEngine On
-
-	# Legacy rewrite from sapphire/ to framework/ namespace
-	RewriteCond %{REQUEST_FILENAME} !-f
-	RewriteCond %{REQUEST_URI} !^framework/main\.php
-	RewriteRule ^sapphire/?(.*) http://doc.silverstripe.org/framework/$1 [R=301,L]
-	
-	# DokuWiki rewrite rules: Need to happen before other rules in order to redirect /doku.php?id=<pagename>
-	# to /pagename, which can then be matched by the legacy rewrite rules further down
-	RewriteCond %{QUERY_STRING}          ^(\bid\b=([^&]*)&?(.*)?)
-	RewriteRule ^doku.php$                http://doc.silverstripe.org/%2?%3 [R=301,L]
-	
-	# Redirect legacy URLs (assumes we're not redirecting other assets, for performance reasons)
-	# Note: Just works on top-level domains, not if the webroot is in a subfolder
-	RewriteCond %{REQUEST_URI} ^(.*)$
-	RewriteCond %{REQUEST_FILENAME} !-f
-	RewriteCond %{REQUEST_URI} !\.(css|gif|ico|jpg|js|png|swf|txt)$
-	#RewriteRule ^execution-pipeline$ http://doc.silverstripe.org/framework/en/reference/ [R=301,L]
-		
-	RewriteRule ^arraydata$ http://api.silverstripe.org/search/lookup/?q=ArrayData&module=sapphire [R=301,L]
```

Comment:
Last touched 3 years ago... and not actually used on the site.
![screen shot 2016-02-11 at 12 44 36 pm](https://cloud.githubusercontent.com/assets/367847/12965731/48ee70ec-d0bd-11e5-8683-da252d6328a8.png)